libs/cart/src/drivers/magento/errors/transform.ts

```diff
@@ -0,0 +1,26 @@
+import { ApolloError } from 'apollo-client';
+import { DaffCartMagentoErrorMap } from './map';
+
+export const lookupDaffCartError = (code: string) => {
+	return DaffCartMagentoErrorMap[code];
+};
+
+const transformGraphQlError = (error: ApolloError): Error => {
```

Comment:
Could just do this, instead of a switch:
```
if(!error.graphQLErrors.length) return error;
const lookup = lookupDaffCartError(error.graphQLErrors[0].extensions.category);
return lookup ? new lookup(error.message) : error;
```

Reply:
addressed